### PR TITLE
Added Intersection_of_two_linkedlist.java file

### PR DESCRIPTION
**Pull request by kapilkumar2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

added the code for finding the insertion point between two linkedlist in java.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Java/ds/Intersection_of_two_linkedlists.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import java.util.*;
```
Comment (HarshCasper) [inline Java/ds/Intersection_of_two_linkedlists.java:3]: Add a beginner introduction about the algorithm that you are looking to implement.

File: Java/ds/Intersection_of_two_linkedlists.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        Node last = head2;  
+        while (last.next != null) 
+            last = last.next; 
+  
+        last.next = newnode; 
+        return; 
+    }
+    
+    
+    int GetNode() //function to get the intersection point of two linked lists
```
Comment (HarshCasper) [inline Java/ds/Intersection_of_two_linkedlists.java]: Use Comments above the Line of Code. Not beside it.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Make apt README changes as well

File: Java/ds/Intersection_of_two_linkedlists.java (hunk 2 above)
Comment (kapilkumar2001) [reply Java/ds/Intersection_of_two_linkedlists.java]: Okay sir, it's updated now.

File: Java/ds/Intersection_of_two_linkedlists.java [hunk 3]
```diff
@@ -0,0 +1,144 @@
+//given two linked lists. and we have to find intersection oint between these two linkedlist. So Here is the program for finding it.
```
Comment (HarshCasper) [inline Java/ds/Intersection_of_two_linkedlists.java:1]: This doesn't look very formal. Can you please take a look at existing code and make the description more standard. Also use multi-line comments for the purpose.

File: Java/ds/Intersection_of_two_linkedlists.java [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        System.out.print("enter "+n2+" elemets(space separated) :");
+        for(int i=0;i<n2;i++)
+        {
+            //inserting element in head2
+              list.append2(sc.nextInt());          
+        }
+ 
+        System.out.println("The node of intersection is " + list.GetNode());
+    }
+}
```
Comment (HarshCasper) [inline Java/ds/Intersection_of_two_linkedlists.java:144]: Add sample I/O as well.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Add README changes as well.

**Comment by atarax665** [comment]
@kapilkumar2001 Are you still working on this? Please reply to reopen this PR.

**Pull request closed without merging**